src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -260,7 +260,7 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
       ! It is only needed when redi mixing is enabled.
       if ( config_use_redi ) then
 
-      if ( config_Redi_use_N2_based_taper .and. .not. local_config_GM_kappa_lat_depth_variable ) then
+      if ( config_Redi_N2_based_taper_enable .and. .not. local_config_GM_kappa_lat_depth_variable ) then
```

Comment:
@vanroekel can you check this logic? We were trying to separate the Redi and GM flags. From below, we have
```
if (config_GM_closure == 'N2_dependent') then
...
 960             local_config_GM_kappa_lat_depth_variable = .true.
```
otherwise it is false. So if `config_GM_closure == 'N2_dependent'` we _do not_ do this section. Does that still make sense? I suspect it is now incorrect, since we only compute RediKappaScaling below, not anything for GM.

Reply:
Thanks for the catch!  you are exactly right.  I just pushed a fix, the later condition should not be there.